packages/hooks/src/useActiveSymbols.ts

```diff
@@ -0,0 +1,27 @@
+import { useFetch } from '@deriv/api';
+import { useMemo } from 'react';
+
+/** A custom hook to get the list of active symbols. */
+const useActiveSymbols = () => {
```

Comment:
Please take look the issue that "active_symbol" not been updated after switching language, Thanks